lib/rules/max-empty-lines/README.md

```diff
@@ -30,12 +31,12 @@ b {}
 Comment strings are also checked -- so the following is a violation:
 
 ```css
-/**
- * Call me Ishmael.
- *
- *
- *
- * Some years ago -- never mind how log precisely -- ...
+/*
+ Call me Ishmael.
+
+
+
+ Some years ago -- never mind how log precisely -- ...
  */
```

Comment:
Should both forms of comments be displayed here?

i.e:

```
/**
 * Call me Ishmael.
 *
 *
 *
 * Some years ago -- never mind how log precisely -- ...
/*
```

And
```
/*
 Call me Ishmael.



 Some years ago -- never mind how log precisely -- ...
 */
```

Reply:
From my point of view, this comment doesn't have empty lines (please attention to stars):

```
/**
 * Call me Ishmael.
 *
 *
 *
 * Some years ago -- never mind how log precisely -- ...
 */
```

That's why I removed from readme.md.

Reply:
I agree with @GorwinJororis. This example doesn't have empty lines.